test/trento/application/projectors/host_projector_test.exs

```diff
@@ -172,6 +176,10 @@ defmodule Trento.HostProjectorTest do
     projection = Repo.get!(HostReadModel, host_id)
 
     assert cluster_id == projection.cluster_id
+
+    refute_broadcast "host_details_updated",
+                     %{id: ^host_id},
+                     1000
```

Comment:
I'm just a bit worried about this `1000` value.
In the `refute` this blocks the test until the timeout is expired, so it will be waiting for 1 second to have a green test.
Maybe we should use the default `100` value?
@CDimonaco @fabriziosestito Any suggestion on this?

Reply:
I think the 1000 value was set by me when I started doing broadcast test, this was set by testing manually different values, if the default timeout is sufficient we can switch to that

Reply:
I mean, in `assert_broadcast` is fine, because it will just wait until the message is sent. And it will be a long test only on failure.
The `refute` will always wait for the time, even in passing tests.
That's the difference

Reply:
I reduced this timeout to 100 mS 👍

Reply:
Just a least request XD
Could you test if putting this same `refute_broadcast` with 100ms in the test above fails? Of course, just temporarily, in your local env.
It should fail. I want to be sure that the 100ms makes sense

Reply:
Just did on my local machine and `refute_broadcast` can catch messages using a `timeout` as low as 1 mS 😵 I guess we may see different performance in the CI environment though.